randovania/games/samus_returns/logic_database/Area 3 - Factory Interior.txt

```diff
@@ -684,8 +679,7 @@ Extra - asset_id: collision_camera_030
   * Tunnel to Gamma Arena & Transport to Metroid Caverns East/Tunnel to Gamma Arena Center Access (Bottom)
   > Event - Gamma Awakened
       All of the following:
-          Morph Ball
-          Movement (Intermediate) or Climb Rooms Vertically (High Jump)
```

Comment:
What was this movement used for?

Reply:
I literally don;t remember which was why I removed it. :p My best guess is that it might've been for an unmorph jump out of the tunnel. (I seriously need to add more videos. Might just save that for post-stable)